.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
 
     strategy:
       matrix:
-        node: ['10', '12']
+        node: ['12']
```

Comment:
Why drop Node 10?

Reply:
This was actually motivated by an earlier point in my iterations when I was using `require.resolve` with the `paths` argument. That didn't make it into the final PR, so everything still technically works under Node 10 now.

However, the other changes in this PR will already require a semver major release, and Node 10 is only 7 weeks from end-of-life, so I still think dropping it is a good idea. If you would rather not though, we can put it back.

Reply:
If we want to release this as a major, then we can drop, otherwise let's leave it.

Reply:
removed